packages/mermaid/src/diagrams/class/classTypes.ts

```diff
@@ -14,6 +16,95 @@ export interface ClassNode {
   tooltip?: string;
 }
 
+export class ClassMember {
+  id!: string;
+  cssStyle!: string;
+  memberType!: string;
+  visibility!: string;
+  classifier!: string;
+  parameters!: string;
+  returnType!: string;
+
+  constructor(input: string, memberType: string) {
```

Comment:
Should we limit this type so it can only be two different values?

```suggestion
  memberType!: 'method' | 'attribute';
  visibility!: string;
  classifier!: string;
  parameters!: string;
  returnType!: string;

  constructor(input: string, memberType: 'method' | 'attribute') {
```